packages/client/modules/newTeam/components/NewTeamForm/NewTeamForm.tsx

```diff
@@ -129,7 +129,7 @@ const NewTeamForm = (props: Props) => {
   const disableFields = !!lockedSelectedOrg && !isNewOrg
   const selectedOrg = organizations.find((org) => org.id === orgId)
   const selectedOrgTeamMemberEmails = selectedOrg?.teams.flatMap(({teamMembers}) =>
-    teamMembers.filter(({isSelf}) => !isSelf).map(({email}) => email)
+    teamMembers?.filter(({isSelf}) => !isSelf).map(({email}) => email)
```

Comment:
Not sure I understand, in what cases `teamMembers` will be null? I see we are fetching them above in graphql query

Reply:
Looks like the `teamMembers` haven't populated in the newly created team. I've updated the `AddTeamMutation` to query for the `teamMembers` as that's a cleaner way of doing it